docs/3.0/manage/settings-and-profiles.mdx

```diff
@@ -26,7 +25,8 @@ prefect config validate
 
 Prefect profiles are persisted groups of settings on your local machine. 
 By default these settings are stored in a [TOML](https://toml.io/en/) file located at `~/.prefect/profiles.toml`.
-This location can be configured through the setting `PREFECT_PROFILES_PATH`. 
+This location can be configured through the setting `PREFECT_PROFILES_PATH`.
```

Comment:
nit: weakly held preference to simplify phrasing
```suggestion
This location can be configured by setting `PREFECT_PROFILES_PATH`.
```

Reply:
Agree.